addons/web/static/src/views/relational_model/dynamic_group_list.js

```diff
@@ -139,7 +141,15 @@ export class DynamicGroupList extends DynamicList {
             displayName: groupName,
         };
 
-        this.groups.push(this._createGroupDatapoint(data));
+        const group = this._createGroupDatapoint(data);
+        this.groups.push(group);
+
+        if (lastGroup) {
+            const resModel = isRelational(this.groupByField)
```

Comment:
only many2one

Reply:
throw error if called and not grouped by many2one. Do it at both places, at the model entry point